### PR TITLE
CGNS-220

### PR DESCRIPTION
**Pull request by MicK7**

### CODE REVIEW

**Comment by MicK7** [comment]
Let's try to merge the changes before 4.2 release. This is already a good improvement.
Further improvement can always be added after 4.2 release.

File: src/adfh/ADFH.c [hunk 1]
```diff
@@ -2236,6 +2238,13 @@ void ADFH_Database_Open(const char   *name,
     }
   }
 #endif
+  
+  if (0 == strcmp(fmt, "DISKLESS")) {
```
Comment (brtnfld) [inline src/adfh/ADFH.c:2242]: See:

CGNS-239 -- Add diskless option for creating a CGNS file

listing the additional items needed for completion of this feature.

Comment (MicK7) [reply src/adfh/ADFH.c:2242]: Oops did not see that coming. Do you want to keep this feature in the branch to add testing or move it to another branch?

Comment (brtnfld) [reply src/adfh/ADFH.c:2242]: We should probably move it to another branch.

Comment (MicK7) [reply src/adfh/ADFH.c]: done

**Comment by MicK7** [comment]
Do we need to setup a CI github action for linux system to check this topic ?

**Comment by brtnfld** [comment]
I will switch the daily testing that we do to test the branch instead of develop. It should run through all the configurations over the next two days.

**Comment by brtnfld** [comment]
What is branch CGNS-220bis? Is that what I should be testing instead?

**Comment by MicK7** [comment]
> 
> 
> @MicK7 What is branch CGNS-220bis? Is that what I should be testing instead?

I removed CGNS-220bis since the changes where added to CGNS-220.
CGNS-220 is the right branch to test.

**Review by brtnfld — APPROVED** [review]
All the internal DT passed.

**Comment by MicK7** [comment]
@brtnfld Thanks for the testing. I just merged the branch.

**Comment by brtnfld** [comment]
@MicK7 don't delete the branch just yet, I need to do some more testing on it. For some reason the develop branch is failing some of the tests with openmpi.

**Comment by MicK7** [comment]
@brtnfld well received

**Comment by brtnfld** [comment]
@MicK7, the branch can be deleted.

**Comment by MicK7** [comment]
> @MicK7, the branch can be deleted.

Do we re-open a new CGNS-220 branch after 4.2 release to tackle the cg_open issue ?

**Comment by brtnfld** [comment]
Or create a new branch, otherwise, we will have to keep it synced with develop.

**Pull request merged**